### PR TITLE
Add feed-dns docker release

### PR DESCRIPTION
**Pull request by chbatey**
This adds a build for dns

Hard to test the release phase but tested the docker phase locally

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,33 +41,47 @@ test :
...
+	  build_files=$${build}_files; \
+	  build_dir=build/$$build; \
+	  binary=$(GOPATH)/bin/feed-$$build; \
+	  docker_file=Dockerfile_$$build; \
+	  mkdir -p $$build_dir; \
+	  cp $$docker_file $${build_dir}/Dockerfile; \
+	  cp $$binary $$build_dir; \
+	done
+
+	cp ${template} ${ingress_build_dir}
```
Comment (chbatey) [inline Makefile]: I could have added a var with the extra files in and had an additional for loop but that seemed over kill for one file

File: Makefile [hunk 2]
```diff
@@ -41,33 +41,47 @@ test :
...
 docker : copy
-	@echo "== build docker image"
-	docker build -t $(docker_tag) build/.
-	@echo "Built $(docker_tag)"
+	@echo "== build docker images"
+	@for build in $(builds) ; do \
+	  tag=skycirrus/feed-$$build:$(git_rev) ; \
+	  docker build -t $$tag build/$${build}/. ; \
+	  echo "Built $$tag" ; \
+	done
```
Comment (jsravn) [inline Makefile]: I suppose it would be purer to use native makefile "foreach" syntax, rather than using the shell here.

**Comment by jsravn** [comment]
lgtm

File: Makefile (hunk 2 above)
Comment (chbatey) [inline Makefile]: i did look at it, as we have multiple lines to execute it didn't get a little nasty

**Pull request merged**